scripts/download-ffmpeg.js

```diff
@@ -9,7 +9,7 @@ const ora = require('ora');
 
 let spinner = ora({text: 'Installing 7zip', stream: process.stdout}).start();
 
-const FFMPEG_URL = 'https://evermeet.cx/ffmpeg/ffmpeg-85767-gdec2fa8.7z';
+const FFMPEG_URL = 'https://evermeet.cx/pub/ffmpeg/snapshots/ffmpeg-86910-g797c232.7z';
```

Comment:
Sorry for the late reply, @esalter-va. 

Yeah, this has been fixed on master 👌